gitops/kcp/base/plnsvc-manager.yaml

```diff
@@ -0,0 +1,131 @@
+---
```

Comment:
Because the life cycle of these resources are currently tied to one another and having everything in one file makes it easy to show that relationship.

If I had the following files [namespace.default.yaml, kcp-manager.yaml, kcp-syncer.yaml, kcp-syncer-binding.yaml], one could remove kcp-manager.yaml and not know that the other files should be cleaned up as well.

If that's an anti-pattern and I'll split it.

Reply:
Isn't it what kustomization.yaml or Argo CD Application bring?

Reply:
My understanding is that ArgoCD/Kustomize ties all the resources for a single application together, but that it does not allow to group a subset of those resources together.